torch/jit/__init__.py

```diff
@@ -805,6 +817,102 @@ def init_then_register(self, *args, **kwargs):
 
 if _enabled:
     class ScriptModule(with_metaclass(ScriptMeta, torch._C.ScriptModule, Module)):
+        r"""
+        The core data structure in Torch Script is the ``ScriptModule``. It is an analogue
+        of torch's nn.Module and represents an entire model as a tree of submodules.
+        Like normal modules, each individual module in a ScriptModule can have submodules,
+        parameters, and methods. In nn.Modules methods are implemented as Python functions,
+        but in ScriptModules methods are actually *Torch Script* functions, a statically-typed subset
```

Comment:
This seems to imply that they are like that by default which is not the case. They only *can* be script functions

Reply:
I'll clarify. I don't want to get too much into the details of mixing python and script until later.